sklearn/model_selection/tests/test_search.py

```diff
@@ -1334,6 +1336,25 @@ def test_fit_grid_point():
                          {'score': scorer}, verbose=True)
 
 
+# FIXME remove test_fit_grid_point_deprecated as
+# fit_grid_point will be removed on 0.25
+def test_fit_grid_point_deprecated():
+    X, y = make_classification(random_state=0)
+    cv = StratifiedKFold()
+    svc = LinearSVC(random_state=0)
+    scorer = make_scorer(accuracy_score)
+    msg = (
+        "fit_grid_point is deprecated in version 0.23 "
+        "and will be removed in version 0.25")
+    params = {'C': 0.1}
+    train, test = next(cv.split(X, y))
```

Comment:
Nit:

```suggestion
    train, test = next(StratifiedKFold().split(X, y))
```

and remove the cv definition above.

Reply:
done